src/conda_package_handling/conda_fmt.py

```diff
@@ -27,6 +29,9 @@
 ZSTD_COMPRESS_THREADS = 1
 
 
+LOG = logging.getLogger(__file__)
```

Comment:
```suggestion
LOG = logging.getLogger(__name__)
```

Reply:
hah. fixed and also changed in tarball.py, where this originates from